lib/ansible/modules/windows/win_git.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+# Copyright: (c) 2018, Anatoliy Ivashina <tivrobo@gmail.com>, Pablo Estigarribia <pablodav@gmail.com>, Michael Hay <project.hay@gmail.com>, Ripon Banik <ripon.banik@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_git
+author: 
+  - Anatoliy Ivashina
+  - Pablo Estigarribia
+  - Michael Hay
+  - Ripon Banik
+version_added: "2.8"
+short_description: Deploy software (or files) from git checkouts.
+description:
+    - Deploy software (or files) from git checkouts.
+options:
+  repo:
+    description:
+        - git, SSH, or HTTP(S) protocol address of the git repository.
+    required: true
+    aliases: [ name ]
+  dest:
+    description:
+        - The path of where the repository should be checked out. If
+          not specified creates the direcctory under user profile folder.
+    required: false
+  version:
+    description:
+        - What version of the repository to check out.  This can be
+          the literal string C(HEAD), a branch name, a tag name.
+          It can also be a I(SHA-1) hash, in which case C(refspec) needs
+          to be specified if the given revision is not already available.
+    default: "HEAD"
+  remote:
+    description:
+        - Name of the remote.
+    default: "origin"  
+  replace_dest:
```

Comment:
Where possible, generally its a good thing to keep the parameter names the same as the linux module equivalent.
is 'replace_dest' the same thing as 'force' in the git module https://docs.ansible.com/ansible/latest/modules/git_module.html ?  If it is the same it would nice if it has the same name.

Reply:
@jhawkesworth git module force has other functionality, so until all of them implemented just want to keep it